### PR TITLE
Add actual thumbs_ups

### PR DESCRIPTION
**Pull request by jeanconn**
Just an idea for some actual thumbs-up checking.

### CODE REVIEW

File: proseco/core.py [hunk 1]
```diff
@@ -82,6 +82,7 @@ def __init__(self, data=None, print_log=False, **kwargs):
             self._default_formats[name] = '.2f'
         for name in ('ra', 'dec', 'RA_PMCORR', 'DEC_PMCORR'):
             self._default_formats[name] = '.6f'
+        self.thumbs_up = False
```
Comment (jeanconn) [inline proseco/core.py]: Looks l don't really have booleans on the matlab side and the python interface won't pass a numpy.bool type so I end up converting to int anyway.  Should probably just store these as ints on the python side.

File: proseco/catalog.py [hunk 2]
```diff
@@ -54,11 +54,16 @@ def _get_aca_catalog(obsid=0, att=None, man_angle=None, date=None,
     aca.fids = get_fid_catalog(detector=detector, sim_offset=sim_offset,
                                focus_offset=focus_offset, acqs=aca.acqs,
                                stars=aca.acqs.meta['stars'], print_log=print_log)
+    # Set the fid thumbs up to true for now as I'm not sure what to check
```
Comment (taldcroft) [inline proseco/catalog.py]: Thumbs up for fid can be if it `len(fids) == fids.meta['n_fid']`.  Set it in `get_fid_catalog`.

File: proseco/core.py (hunk 1 above)
Comment (taldcroft) [reply proseco/core.py]: As always, leave a vertical line of space between entirely unrelated blocks (or lines) of code.

Comment (taldcroft) [reply proseco/core.py]: Add a `thumbs_up` property which sets onto the same-named meta key.
```
@thumbs_up.setter
def thumbs_up(self, val):
    self.meta['thumbs_up'] = int(val)  # Coerce to int for interface with Matlab
```

File: proseco/characteristics.py [hunk 3]
```diff
@@ -97,3 +97,7 @@
                  1196953168,
                  1197635184]
 
+# Minimum acquisition probability thresholds from starcheck
+# for thumbs_up
```
Comment (taldcroft) [inline proseco/characteristics.py]: Doesn't need to be split into two lines.

**Comment by taldcroft** [comment]
Oops, I forgot to add that it needs tests.

File: proseco/core.py (hunk 1 above)
Comment (jeanconn) [reply proseco/core.py]: I thought thumbs_up already was a property set to the same-named meta key?  Wouldn't this just be a change to for the type?

Comment (taldcroft) [reply proseco/core.py]: Indeed...

**Comment by taldcroft** [comment]
Not sure if you deleted that previous comment, but yes there is an `n_fid` in the PR (though it is mistakenly called `n_fids`).  But for this PR maybe just stub the fid thumbs up to True (i.e. ignore my previous suggestion)?

**Comment by jeanconn** [comment]
Which PR has n_fid or n_fids?  I'm not seeing where this is handled.  (fine to go back to just setting it thumbs_up true for fid catalog but trying to be situationally aware on progress and interface).

**Comment by jeanconn** [comment]
Gotcha. Yes, for integration I wasn't sure if get_aca_catalog needed to pass n_fid to get_acq... to then have it get in acq meta etc... but I think that doesn't drive any changes to my top level API (as long as I can assume that we only have the requested fids by the time it gets to the matlab interface).

**Pull request merged**